eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -5,21 +5,54 @@ parameters:
 stages:
   - stage: Build
     jobs:
-    - template: ../jobs/archetype-sdk-data.yml
+    - template: ../jobs/archetype-sdk-client.yml
       parameters:
         ServiceDirectory: ${{parameters.ServiceDirectory}}
         SDKType: ${{parameters.SDKType}}
         Artifacts: ${{parameters.Artifacts}}
         TestMatrix:
-          Windows - java8:
+          Linux - Java 8:
```

Comment:
It's worth mentioning here that this test matrix, for the unit tests, brings Cosmos v4 inline with the test matrix used for client track (Linux/Mac/Windows with Java versions 8 and 11). The "from source" build is a special run that was added a while back that manipulates the versions such that all track client track dependencies are built from the latest source. This is a safeguard to ensure compatibility with changes being made to inter track dependencies.